src/main/java/com/ticketbooking/exception/TicketBookingNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.ticketbooking.exception;
+
+public class TicketBookingNotFoundException extends RuntimeException {
```

Comment:
Czemu Runtime?

Reply:
Jest to narzucone porzez mechanizm z annotacja @RestControllerAdvice, inaczej by to nie zadziałało